kitsune/sumo/static/sumo/js/wiki.js

```diff
@@ -638,6 +639,27 @@
     }
   }
 
+  function initTranslationDraft() {
+    var $draftButton = $('.btn-draft'),
+      url = $('.btn-draft').data('draft-url'),
+      draftMessage = $('#draft-message-success').html();
+    $draftButton.click( function() {
+      $('#draft-message-success').hide()
+      $('#draft-message-saving').show()
+      var bothData = $('#both_form').serializeArray(),
+        docData = $('#doc_form').serializeArray(),
+        revData = $('#rev_form').serializeArray(),
+        totalData = $.extend(bothData, docData, revData);
+      $.post(url, totalData)
+        .done(function() {
+          var time = new Date(),
+            message = interpolate(gettext('<strong>Draft has been saved on:</strong> %s'), [time]);
+          $('#draft-message-success').html(message).show();
+          $('#draft-message-saving').hide();
+        });
```

Comment:
You should add a `.fail(function() { ... })` handler to this to show an error message in case the server can't be contacted.